packages/databyss-ui/components/PageContent/IndexPageContent.js

```diff
@@ -1,9 +1,18 @@
 import React from 'react'
 import { Text, View, ScrollView } from '@databyss-org/ui/primitives'
-import { pxUnits } from '@databyss-org/ui/theming/views'
+import { pxUnits, widthVariants } from '@databyss-org/ui/theming/views'
 
 const IndexPageContent = ({ title, children }) => (
-  <ScrollView p="medium" flex="1" maxHeight="98vh">
+  <ScrollView
+    p="medium"
+    flex="1"
+    maxHeight="98vh"
+    maxWidth={widthVariants.content.maxWidth}
```

Comment:
why not just use `widthVariant="content"`?

Reply:
it didn't work, it just translated the css to "max-width: content" and not "max-width: 700px"

Reply:
Actually I tried it again and strangely it works now... maybe I passed the `maxWidth` prop before instead of `widthVariant`, I don't know